.buildlibrary

```diff
@@ -1,11 +1,12 @@
-ValidationKey: '227436669'
+ValidationKey: '227587360'
 AcceptedWarnings:
 - 'Warning: package ''.*'' was built under R version'
 - 'Warning: namespace ''.*'' is not available and has been replaced'
 - .*qpdf.* is needed for checks on size reduction of PDFs
 - .*following variables are expected in the piamInterfaces.*
 - Summation checks have revealed some gaps.*
 - .*to check if summation groups add up.*
+- .* Negative numbers in weight. Dangerous, was it really intended?.*
```

Comment:
Which calculation returns this? Seems to be using `madrat::toolAggregate()`.

Reply:
I will look into this separately and try to get rid of the warning again.I asked Gabriel to add it here so our checks don't hinder his own work. Should not block merging this.